Source/BestProjectEver/Private/Character/BPE_Enemy.cpp

```diff
@@ -111,6 +116,16 @@ void ABPE_Enemy::OnRep_ColorType()
 	UpdateMeshColor();
 }
 
+//----------------------------------------------------------------------------------------------------------------------	
+void ABPE_Enemy::FollowSplinePath_Implementation(const FVector& NextPointLocation)
+{
+	if(IsValid(GetMovementComponent()))
+	{
+		const FRotator LookAtRotation = UKismetMathLibrary::FindLookAtRotation(GetActorLocation(), NextPointLocation);
+		AddMovementInput(UKismetMathLibrary::GetForwardVector(LookAtRotation), GetMovementComponent()->GetMaxSpeed(), true);	
```

Comment:
Quick tip just so you know, but this way is perfectly fine. `LookAtRotation.Vector()` does the same thing as that long Kismet Math Library method.